monai/bundle/scripts.py

```diff
@@ -321,3 +325,85 @@ def verify_net_in_out(
         if output.dtype != output_dtype:
             raise ValueError(f"dtype of output data `{output.dtype}` doesn't match: {output_dtype}.")
     logger.info("data shape of network is verified with no error.")
+
+
+def export(
```

Comment:
Could we call this `checkpoint_export` instead to differentiate from other places functions/methods of this name are used? I would suggest adding another function `net_export` to take a network instance and do the exporting with that so that people have something they can call in their own code with the network they just finished training. We could also have a third function which takes a Torchscript file instead of a checkpoint file which could copy it into the bundle and extract the checkpoint file from it to save separately. These latter extensions we could leave for later

Reply:
Hi @ericspod ,

I changed it to `ckpt_export` in the latest commit.
About the TorchScript input file idea, how can we automatically determine it's a checkpoint or torchscript file?

Thanks.

Reply:
I feel it's a little bit confusing to load `TorchScript` model as "checkpoint" in this function, maybe we can add a separate function for "convert TorchScript model" later?

Thanks.